docs/glossary.md

```diff
@@ -31,3 +31,19 @@
   - Defending units can only participate in this phase if `Submarines Defending May Submerge Or Retreat` property is set.
   - `Sub Retreat Before Battle` defines if this phase occurs before the First Strike Phase or at the end of the Battle Round.
 - **Submerge Subs Phase**: Another name for **Evasion Phase** and generally used when only submarines have `canEvade` and `Submersible Subs` is set.
+
+##Development Terms
+- **Branch**: A Branch is a copy of the main codebase.  It is used to keep changes separate until they are completed or in a stable state.  
+- **Checkout**: When you Checkout a branch, that is where commits are stored.  
+- **Clone**: Cloning is when you take your repository and copy it to your local machine to work on.  
+- **Commit**: A Commit is a list of changes being made to a repository.  
```

Comment:
This definition seems imprecise. A commit is typically viewed as an atomic entity that itself is a set of file differences.